addon/components/o-s-s/avatar.ts

```diff
@@ -0,0 +1,61 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+type SizeType = 'xs' | 'sm' | 'md' | 'lg';
+type SizeDefType = { [key in SizeType]: string };
+export const SizeDefinition: SizeDefType = {
+  xs: 'upf-avatar--xs',
+  sm: 'upf-avatar--sm',
+  md: 'upf-avatar--md',
+  lg: 'upf-avatar--lg'
+};
+
+interface OSSAvatarArgs {
+  image?: string;
+  initials?: string;
+  size?: SizeType;
+}
+
+export const DEFAULT_IMAGE_URL: string = '/assets/images/upfluence-white-logo.svg';
+
+export default class OSSAvatar extends Component<OSSAvatarArgs> {
+  @tracked hasError: boolean = false;
+  @tracked displayInitials: boolean = false;
+
+  constructor(owner: unknown, args: OSSAvatarArgs) {
+    super(owner, args);
+    if (!args.image && args.initials) this.displayInitials = true;
+  }
+
+  get image(): string | undefined {
+    return !this.args.image && !this.args.initials ? DEFAULT_IMAGE_URL : this.args.image;
```

Comment:
```suggestion
    return this.args.image ?? DEFAULT_IMAGE_URL;
```
Do you really need to keep this after added that `if (!args.image && args.initials) this.displayInitials = true;` in the constructor